src/routes/invidious_routes/dashManifest.ts

```diff
@@ -77,6 +79,13 @@ dashManifest.get("/:videoId", async (c) => {
                 } else {
                     return false;
                 }
+            }).filter((i) => {
```

Comment:
Why a second filter. Why not in the first filter?

Reply:
This should be better. I went the easy way in the first commit